src/main/java/com/microsoft/cse/helium/app/controllers/RobotsController.java

```diff
@@ -0,0 +1,43 @@
+package com.microsoft.cse.helium.app.controllers;
+
+// import com.microsoft.applicationinsights.core.dependencies.apachecommons.io.IOUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+ 
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+/*  
+@RestController
+public class RobotsController {
+    @RequestMapping(value={"/robots*.txt"}, method = RequestMethod.GET)
```

Comment:
this needs to handle robots*.txt to handle the App Service startup request which adds a random into - robots1234.txt

Reply:
That will be implemented when I return to robots. This PR is not for robots work.